.github/workflows/black.yml

```diff
@@ -10,8 +10,8 @@ jobs:
   check-black-formatting:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
-    - uses: actions/setup-python@v2
+    - uses: actions/checkout@v4
+    - uses: actions/setup-python@v4
```

Comment:
I think this does not belong to the PR here. Is this needed for the PR? I would do a separate PR for this, and then rebase this one here.

Reply:
https://github.com/rwth-i6/i6_models/pull/52